spec/features/map/gibs_collection_visualization_spec.rb

```diff
@@ -1,63 +1,43 @@
-# EDSC-188 As a user, I want to view GIBS tile imagery corresponding to my
-#          search area on a map so that I may preview my results
-# EDSC-193 As a user, I want to see information about which collections have GIBS
-#          support so I may understand why some collections render differently
-
 require 'rails_helper'
 
-describe "Collection GIBS visualizations", pending_updates: true do
+describe 'Collection GIBS visualizations' do
   extend Helpers::CollectionHelpers
 
-  gibs_collection_id = 'C119124186-NSIDC_ECS'
-  gibs_collection_name = 'AMSR-E/Aqua L2B Global Swath Rain Rate/Type GSFC Profiling Algorithm V002'
-  gibs_tile_layer = '.leaflet-tile-pane .leaflet-layer:nth-child(2)'
+  gibs_collection_id = 'C1443528505-LAADS'
 
-  context "when viewing a GIBS-enabled collection in the results list" do
-    before :all do
-      load_page :search, authenticate: 'edsc'
-      fill_in "keywords", with: gibs_collection_id
+  context 'when viewing a GIBS-enabled collection in the results list' do
+    before do
+      load_page :search
+      fill_in 'keywords', with: gibs_collection_id
       wait_for_xhr
-      expect(page).to have_content('AE_Rain')
     end
-    it "indicates that the collection has GIBS visualizations" do
+
+    it 'indicates that the collection has GIBS visualizations' do
       expect(first_collection_result).to have_css('.badge-gibs')
     end
-    after :all do
-      reset_search
-      wait_for_xhr
-    end
   end
 
-  context "when visualizing a GIBS-enabled collection" do
+  context 'when visualizing a GIBS-enabled collection' do
     before :all do
-      load_page :search, authenticate: 'edsc'
-      fill_in "keywords", with: gibs_collection_id
+      load_page :search
+      fill_in 'keywords', with: gibs_collection_id
```

Comment:
Can avoid the separate methods by sending `q: gibs_collection_id` to `load_page`, I think.